src/decidim/electionguard/trustee.py

```diff
@@ -59,8 +59,8 @@ def process_message(self, message_type: str, message: dict, context: Context):
             }
 
 
-class ProcessTrusteesPartialElectionKey(ElectionStep):
-    message_type = 'trustee_partial_election_key'
+class ProcessTrusteesPartialElectionKeys(ElectionStep):
+    message_type = 'trustee_partial_election_keys'
```

Comment:
I've changed the subtype for this message to be consistent with the previous message (`trustee_election_keys`). It shouldn't affect anything else, but just be aware of this.